protokube/pkg/protokube/etcd_manifest.go

```diff
@@ -25,6 +25,16 @@ import (
 
 // BuildEtcdManifest creates the pod spec, based on the etcd cluster
 func BuildEtcdManifest(c *EtcdCluster) *v1.Pod {
+
+	// TODO I do not like these here
```

Comment:
Might be clearer & cleaner if the default flag value was `gcr.io/google_containers`.